### DISCUSSION TITLE
View Archives and Unarchive

### DISCUSSION THREAD

**Discussion by aqib-m31** [iSoron/uhabits#2030] [Feature Requests] [answered]
I accidentally archived something and wanted to restore it back but I couldn't find the the **archives** anywhere and there weren't any settings for **unarchiving**.
I had to manually fix it by firstly making a full export backup and update the 'archived' field in the DB and import it back.

**Comment by KyleSCraig** [comment, marked as answer]
You go into the filters and untick "Hide archived", then you can long press habits, press the three dots and Unarchive
